docs/register-phase-1-stake.md

```diff
@@ -0,0 +1,154 @@
+# Register your Phase-1 stake to Phase-2 through CLI
+
+## Understanding the registration process
+
+To register your Bitcoin stake on the Babylon chain, you must have a confirmed 
+Bitcoin staking transaction. Before proceeding, please review the 
+[Registration of Existing Stakes to the Babylon Chain](https://github.com/babylonlabs-io/babylon/blob/main/docs/stake-registration.md) 
+documentation to check your eligibility and understand the registration 
+requirements and process.
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Setup and configuration](#setup-and-configuration)
+- [Register your stake](#register-your-stake)
+- [Verify your registration](#verify-your-registration)
+- [Receiving rewards](#receiving-rewards)
+
+## Prerequisites
+
+To register your stake to the Babylon blockchain, the following are required:
+- Access to the Bitcoin wallet you have used to create your stake on Bitcoin 
+  (needed to sign transactions).
+- The Bitcoin block height in which your staking transaction was included, which 
+  will be used in the registration command.
+- A funded Babylon address, which will be used to receive the rewards from the 
+  registration.
+- Global parameter file as seen here https://github.com/babylonlabs-io/networks/blob/main/bbn-test-5/parameters/global-params.json 
+  that were used when you created your Bitcoin stake. This will be used in the 
+  registration command later on.
+- A running instance of the `stakerd` daemon. Please follow the `stakerd` 
+  [setup guide](../README.md#3-btc-staker-installation) to complete the setup. 
+  This is also inclusive of creating a Babylon keyring with funds step as the 
+  `stakerd` daemon requires a funded keyring to pay for the transactions.
```

Comment:
```suggestion
- **Staker Daemon**: A running instance of the `stakerd` daemon.
   - **Bitcoin Wallet**: The `stakerd` daemon should connect and have access to the Bitcoin wallet
     that you have previously used to create your phase-1 stake on Bitcoin. This will be needed to
     create a proof of ownership of the stake.
   - **Funded Babylon Account**: The `stakerd` daemon should have access to a funded
      Babylon account. This account will pay for transaction fees for the interaction with
      the Babylon blockchain, as well as be used as the recipient for BTC staking rewards
      in the respective native PoS token (note: testnet tokens have no value).
   - **Setup Guide**: The  `stakerd`
      [setup guide](../README.md#3-btc-staker-installation) contains all details on how
      to set up the `stakerd` and connect it with your Bitcoin wallet and Babylon account.
- **Phase-1 Parameters**: The staker CLI requires the parameters that were used when creating
   the phase-1 staking transaction in order to construct a valid Babylon chain registration transaction.
   Specifically, it requires the following:
   - **Global Parameter Versions**: The global parameters file containing the full parameters versions for
      the phase-1 network you submitted your stake in (e.g.,
      [testnet parameters](https://github.com/babylonlabs-io/networks/blob/main/bbn-test-4/parameters/global-params.json).
   - **Bitcoin Inclusion Height**: The Bitcoin block height in which your staking transaction was included, which
      will be used in the registration command. You can find this through Bitcoin explorer
      tools such as [mempool.space](https://mempool.space).
```